### DISCUSSION TITLE
Is there any way to use csv/hdf5 files as checkpoint?

### DISCUSSION THREAD

**Discussion by mojtabaAmiraslanpour** [UvaCsl/HemoCell#54] [General discussion]
Hello all,

So I have a critical issue here. Due to disk space error, my checkpoint files are written with zero bytes, and I cannot continue my two-week simulation run. Is there any way I can make checkpoint files again using the existing HDF5/CSV files?

Any help is appreciated

**Comment by gzavo** [comment]
Hi @mojtabaAmiraslanpour,

Sadly that is unlikely. For velocity, the x,y,z components are saved, but LBM uses more degrees of freedom (19 instead of 3). While it is possible to reconstruct the equilibrium values, the exact same state of the simulation cannot be reconstructed from the HDF5 output, only the checkpoint file.
